### DISCUSSION TITLE
Help – Need OG np.Templating Web Services Script

### DISCUSSION THREAD

**Discussion by robynshepard** [NotePlan/plugins#454] [Q&A]
I am dumb and accidentally pasted over the original script in web services under the settings. can someone please give me what the original scripts were that were set when it was downloaded? I cannot even restore from Time Machine fml

**Comment by dwertheimer** [comment]
If you just delete, the nP.Templating folder, you can download a fresh copy from the Preferences plug-in panel.

**Reply by robynshepard** [reply]
I tried this and I dont know if its because it is synced with my ipad/phone, but it doesn't work :( it keeps reloading with the script I pasted over

**Reply by robynshepard** [reply]
additionally, the red delete button is unresponsive :(

**Reply by dwertheimer** [reply]
@EduardMe you have other ideas?

**Reply by EduardMe** [reply]
Sounds like you need to delete the settings or data folder inside the plugin folder? 

![image](https://github.com/NotePlan/plugins/assets/3285090/b376fe41-986d-4d11-91da-ca332d15c2e9)

**Comment by robynshepard** [comment]
it worked!!! thank you so much!
